app/routes/docs_.schema.$version.$/route.tsx

```diff
@@ -52,7 +52,6 @@ export async function loader({
 
 export default function () {
   const { version, '*': path } = useParams()
-  // const [showVersions, setShowVersions] = useState(false)
```

Comment:
There's still a line of commented out code here.

Reply:
That line should be gone now, just pushed a new commit